VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -68,7 +86,78 @@ public void deleteProfileImage(@NonNull String username) throws RetroHttpExcepti
         return userService.getBadges(username, page, ApiConstants.STANDARD_PAGE_SIZE);
     }
 
-    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username) throws RetroHttpException {
-        return userService.getUserEnrolledCourses(username);
+    public List<EnrolledCoursesResponse> getUserEnrolledCourses(@NonNull String username, boolean tryCache) throws RetroHttpException {
+        List<EnrolledCoursesResponse> ret = null;
+        String json = null;
+
+        final String cacheKey = config.getApiHostURL() + "/api/mobile/v0.5/users/" + username + "/course_enrollments";
```

Comment:
You could remove it from `Api.java` and define it somewhere else, called by `Api.java` Perhaps have it call `UserAPI`. Or perhaps move it to `ApiConstants` and have them both use that..